src/actions/file-search.js

```diff
@@ -189,5 +193,6 @@ export function closeFileSearch(editor: Editor) {
     dispatch(setFileSearchQuery(""));
     dispatch(closeActiveSearch());
     dispatch(clearHighlightLineRange());
+    dispatch(clearSearchResults());
```

Comment:
I think this will also affect closing the SearchBar so I can change this if you don't won't the results cleared when the search bar is closed.

Reply:
maybe we should just clear the search results when we call `setFileSearchQuery`? It is nice to keep the number of discrete actions down..

Reply:
I just realized that the reducers are now listening for the NAVIGATE action to do this. So I'll just take this out.